DICOM/.editorconfig

```diff
@@ -0,0 +1,11 @@
+﻿[*.cs]
+
+# CS3021: Type or member does not need a CLSCompliant attribute because the assembly does not have a CLSCompliant attribute
+dotnet_diagnostic.CS3021.severity = none
+
+
+# CS0618: Type or member is obsolete
+dotnet_diagnostic.CS0618.severity = none
+
+# xUnit1026: Theory methods should use all of their parameters
+dotnet_diagnostic.xUnit1026.severity = none
```

Comment:
Do we remove these warnings intentionally?

Reply:
Can we suppress this warning?

Reply:
Seems in most cases the rules are correct. We can suppress it specifically in the code.

Reply:
Removed it here.